VRCFaceTracking/Params/Expressions/Legacy/Lip/LipShapeMerger.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using VRCFaceTracking.Params.Lip;
+
+namespace VRCFaceTracking.Params.LipMerging
+{
+    [Obsolete("SRanipal Lip Shapes are being phased out. Please switch to Unified Expressions.")]
```

Comment:
I noticed that this was tagged as Obsolete, generally module devs don't touch this class when sending tracking data to external modules anyhow. @benaclejames what do you think about labelling the other SRanipal classes (such as the SRanipal enum, and shape transforms of SRanipal) and shapes as Obsolete?

Reply:
Hmm, provided they're within the VRCFT namespace, that sounds like a good idea!

Reply:
Ah that makes sense, given that most of the SRanipal stuff was either removed as dependencies or replaced with a VRCFT namespaced version (specifically SRanipal_LipShape_v2 in UnifiedExpressions.cs), this specific class being obsoleted makes sense. I think that clears up everything!